code/game/objects/objs.dm

```diff
@@ -307,12 +307,12 @@ a {
 		return TRUE
 	return FALSE
 
-/obj/water_act(volume, temperature, source, method = REAGENT_TOUCH)
+/obj/water_act(volume = 0, temperature = 0, source, method = REAGENT_TOUCH)
```

Comment:
These having defaults doesn't really fit I think

Reply:
that was just me being overzealous. Fixed